root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1787,3 +1776,118 @@ test('Cleanup', function (t) {
 
     t.end();
 });
+
+test('URL cleanup component: auto-select, clean-up, and validation', {}, function (t) {
+    const test_data = [
+        // AllMusic
+        {
+            i_url: 'http://www.allmusic.com/album/here-comes-the-sun-mw0002303439/releases',
+            i_ent: 'release_group',
+            match: 'allmusic',
+            clean: 'http://www.allmusic.com/album/mw0002303439',
```

Comment:
I avoided to use the term _output_ because both `match` and `clean` are actually used as input by the two last tests.
I tried to keep tests conciseness and to add self-explanatory property names (but it looks like I failed that final point).